grammars/ruby.cson

```diff
@@ -360,6 +360,8 @@
         0[oO]?[0-7](?>_?[0-7])*
         |
         0[bB][01](?>_?[01])*
+        |
+        0[dD]\\d(?>_?\\d)*                          # A base indicator can only be used with an integer
```

Comment:
I could instead add a `(?:0[dD])?` to the beginning of L354, but in this way right now, when people write something like `0d1.2`, they could get a hint that it's not a valid number.